lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+      sunday: '$t(ordinals:{{week}}, capitalize) Sunday of Lent',
+      day_after_ash_wed: '$t(weekdays:{{dow}}, capitalize) after Ash Wednesday',
+      holy_week_day: '$t(weekdays:{{dow}}, capitalize) of Holy Week',*/
+    },
+
+    paschal_triduum: {
+      //season: 'Paschal Triduum',
+    },
+
+    easter_time: {
+      season: 'Húsvéti idő',
+      /*weekday: '$t(weekdays:{{dow}}, capitalize) of the $t(ordinals:{{week}}) week of Easter',
+      sunday: '$t(ordinals:{{week}}, capitalize) Sunday of Easter',
+      octave: '$t(weekdays:{{dow}}, capitalize) within the Octave of Easter',*/
+    },
+  },
+
+  periods: {
+    /*epiphany: 'Epiphany',
+    holy_week: 'Holy Week',*/
+  },
+
+  ranks: {
+    //solemnity: 'solemnity',
+    sunday: 'vasárnap',
+    feast: 'ünnep',
+    memorial: 'emléknap',
+    weekday: 'hétköznap',
+  },
+
+  cycles: {
+    //proper_of_time: 'Proper of Time',
+    //proper_of_saints: 'Proper of Saints',
+    sunday_year_a: 'A év',
+    sunday_year_b: 'B év',
+    sunday_year_c: 'C év',
+    weekday_year_1: 'I. év',
+    weekday_year_2: 'II. év',
+    /*psalter_week_1: 'Week I',
+    psalter_week_2: 'Week II',
+    psalter_week_3: 'Week III',
+    psalter_week_4: 'Week IV',*/
+  },
+
+  weekdays: {
+    0: 'vasárnap',
+    1: 'hétfő',
+    2: 'kedd',
+    3: 'szerda',
+    4: 'csütörtök',
+    5: 'péntek',
+    6: 'szombat',
+  },
+
+  months: {
+    0: 'január',
+    1: 'február',
+    2: 'március',
+    3: 'április',
+    4: 'május',
+    5: 'június',
+    6: 'július',
+    7: 'augusztus',
+    8: 'szeptember',
+    9: 'október',
+    10: 'november',
+    11: 'december',
+  },
+
+  colors: {
+    black: 'fekete',
+    gold: 'arany',
+    green: 'zöld',
+    purple: 'lila',
+    red: 'piros',
+    rose: 'rózsaszín',
+    white: 'fehér',
+  },
+
+  ordinals: {
+    '1': 'első',
+    '2': 'második',
+    '3': 'harmadik',
+    '4': 'negyedik',
+    '5': 'ötödik',
+    '6': 'hatodik',
+    '7': 'hetedik',
+    '8': 'nyolcadik',
+    '9': 'kilencedik',
+    '10': 'tizedik',
+    '11': 'tizenegyedik',
+    '12': 'tizenkettedik',
+    '13': 'tizenharmadik',
+    '14': 'tizennegyedik',
+    '15': 'tizenötödik',
+    '16': 'tizenhatodik',
+    '17': 'tizenhetedik',
+    '18': 'tizennyolcadik',
+    '19': 'tizenkilencedik',
+    '20': 'huszadik',
+    '21': 'huszonegyedik',
+    '22': 'huszonkettedik',
+    '23': 'huszonharmadik',
+    '24': 'huszonnegyedik',
+    '25': 'huszonötödik',
+    '26': 'huszonhatodik',
+    '27': 'huszonhetedik',
+    '28': 'huszonnyolcadik',
+    '29': 'huszonkilencedik',
+    '30': 'harmincadik',
+    '31': 'harmincegyedik',
+    '32': 'harminckettedik',
+    '33': 'harmincharmadik',
+    '34': 'harmincnegyedik',
+  },
+
+  names: {
+    //'205_blessed_martyrs_of_japan': '205 Blessed Martyrs of Japan',
+    adalbert_of_prague_bishop: 'Szent Adalbert püspök és vértanú',
+    adalbert_of_prague_bishop_patron_of_poland: 'Szent Adalbert püspök és vértanú, Legyelország patrónusa',
```

Comment:
According to [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=163), in Hungarian, you use _(fő-) pátrónája_ instead of `patrónusa`.

Just a note: in the key name (e.g. `adalbert_of_prague_bishop_patron_of_poland`), we use `patron`/`patroness` for principal patrons (`főpátrón`) and `copatron`/`copatroness` for `??? másodlagos pátrón` (I am not sure if _másodlagos_ is the word for secondary patron in Hungarian).

This should be changed (unless the Hungarian Roman Missal contradicts this) in all places, not just in this key.